### ISSUE TITLE
Autoloading from root with custom namespace

### ISSUE THREAD

**Issue by chiefGui** [php-fig/fig-standards#231]
Hi!

Is there a way to autoload from root with a custom namespace like "MyApp\MyFolder\Class"?

I mean, my application isn't inside a separated and isolated folder that I can simply "$loader->addNamespace('MyApp', '/myApplication/');".

In my case, the "Initiliazer.php" that calls the autoloader is in the root's directory of the application, sharing space with other important folders that contains classes that I need to invoke with autoloader, like Helpers and Controllers.

In other words, what I want is:
"$loader->addNamespace('MyApp', '/'); // (use MyApp\Helpers\Currency should work)
$loader->addNamespace('Helpers', '/Helpers'); // (use Helpers\Currency should work too)"

Any ideas?

Anything that I'm trying is working. Isn't autoloader prepared by deufalt, is it?

(To summarize: I want add namespace for each folder of my application. How can I do this?)
